Doc 1671/archive v1 onboarding docs to preview as reference for v2 gap analysis (#607)
swan-io/docs: docs/preview/new-onboarding/legacy-onboarding/company/index.mdx

---
title: Company account onboarding
---

# Company account onboarding

:::caution Deprecated
This page documents the v1 onboarding API. It is preserved as a read-only reference. Refer to [account onboarding enhancements](../../index.mdx) for the current v2 API.
:::

import CompanyOnboardingDefinition from '../partials/_onboarding-company.mdx';

> <CompanyOnboardingDefinition />
> **Self-employed** users hold company accounts.
Refer to the country requirements section for more information about identification for self-employed users.

## Process overview {#overview}

Company onboarding supports different types of organizations, including:

- Several types of companies
- Self-employed workers
- Associations or not-for-profit organizations

import OnboardingOverview from '../partials/_overview.mdx';

<OnboardingOverview accountType="a company" />

## Collecting onboarding information {#collect-info}

Company onboarding information can be submitted **with the API** or through **Swan's onboarding form**.

| Method | Details |
| --- | --- |

> | Onboarding form | The onboarding form collects:<br /><br /><ol><li>User's **email address**.</li><li>Legal information about the organization, including the **company name**, **registration number** (optional), **VAT number** (optional), and **address**, as well as details about the organization's **activity** and **expected monthly volume**.</li><li>Information about the company's **ultimate beneficial owners** (UBO). This step only applies if the user selected **company**, **home owner association**, or **other**, when submitting their preliminary details.</li><li>Information about the company's **representatives**. This includes individuals or legal entities authorized to act on behalf of the company.</li><li>**Required documents** to verify your organization. You might not see this step in your onboarding process.</li><li>User [**signs up**](../../../../topics/users/index.mdx#signup) for Swan.</li></ol>You can also customize this form to better meet your needs. Review Swan's [open source documentation](https://swan-io.github.io/swan-partner-frontend/specs/onboarding/company/), then ask your PIM (Product Integration Manager) for more information. |

Reviewer: Link to the proof of company reg content.

Reviewer: User signs up for Swan doesn't follow the flow of the bulleted list. i.e. "The onboarding form collects...user signs up for Swan"

Reviewer: The "open source documentation" - who maintains that? Isn't docs.swan.io the open source documentation? 

Also - not all Partners have a PIM.

Author: 1. Out of scope for this archiving PR. Happy to open a follow-up ticket for the live docs
2. Fixing
3. This refers to the open source partner frontend repo, not docs.swan.io. The "ask your PIM" wording is pre-existing in the live docs and out of scope here. Happy to track a fix separately with all the other "legacy docs enhancements", but I believe this shouldn't be a top priority. Happy to discuss.

Reviewer: Who maintains that repo, though, and what extra information lives in that that we need to link to it?

Author: That's swan-io/swan-partner-frontend, it documents the customisable onboarding form UI for partners using Swan's no-code frontend. It's only relevant for partners building on top of that, not for API-only integrations. The reference can stay but could be scoped better.

> | Onboarding form | The onboarding form collects:<br /><br /><ol><li>User's **email address**.</li><li>Legal information about the organization, including the **company name**, **registration number** (optional), **VAT number** (optional), and **address**, as well as details about the organization's **activity** and **expected monthly volume**.</li><li>Information about the company's **ultimate beneficial owners** (UBO). This step only applies if the user selected **company**, **home owner association**, or **other**, when submitting their preliminary details.</li><li>Information about the company's **representatives**. This includes individuals or legal entities authorized to act on behalf of the company.</li><li>**Required documents** to verify your organization. You might not see this step in your onboarding process.</li><li>User [**signs up**](../../../../topics/users/index.mdx#signup) for Swan.</li></ol>You can also customize this form to better meet your needs. Review Swan's [open source documentation](https://swan-io.github.io/swan-partner-frontend/specs/onboarding/company/), then ask your PIM (Product Integration Manager) for more information. |

Author: ```suggestion
| Onboarding form | The onboarding form collects:<br /><br /><ol><li>User's **email address**.</li><li>Legal information about the organization, including the **company name**, **registration number** (optional), **VAT number** (optional), and **address**, as well as details about the organization's **activity** and **expected monthly volume**.</li><li>Information about the company's **ultimate beneficial owners** (UBO). This step only applies if the user selected **company**, **home owner association**, or **other**, when submitting their preliminary details.</li><li>Information about the company's **representatives**. This includes individuals or legal entities authorized to act on behalf of the company.</li><li>**Required documents** to verify your organization. You might not see this step in your onboarding process.</li></ol>After completing the form, the user signs up for Swan. You can also customize this form to better meet your needs. Review Swan's [open source documentation](https://swan-io.github.io/swan-partner-frontend/specs/onboarding/company/), then ask your PIM (Product Integration Manager) for more information. |
```

If you use Swan's onboarding form, this information is collected by default.
If you design your own form, please offer the following ranges:

- Less than 10 000€ (`LessThan10000`)
- Between 10 000€ and 50 000€ (`Between10000And50000`)
- Between 50 000€ and 100 000€ (`Between50000And100000`)
- More than 100 000€ (`MoreThan100000`)

### Acquisition channel {#acquisition-channel}

Use the optional `acquisitionChannel` field to indicate how the account holder was acquired.

| Value | Description |
|-------|-------------|
| `InboundOrganic` | Account holders who signed up directly, such as through your website. |
| `InboundReferral` | Account holders acquired through referrals from existing users or partners. |
| `OutboundSales` | Account holders acquired through direct sales efforts. |
| `OutboundAccountingFirm` | Account holders acquired through accounting firm partnerships. |

Choose the value that best reflects your acquisition method for each account holder.

### Ultimate beneficial owners (UBOs) {#ubo}

Providing information for at least one UBO is **mandatory for all companies**.
Swan allows two types of UBOs:

- Ownership of capital UBO
- Legal representative UBO

> #### Ownership of capital UBO {#ubo-owner}

Author: ```suggestion
#### Definition of a UBO {#ubo-owner}
```

> #### Ownership of capital UBO {#ubo-owner}

Reviewer: Why do we reference as this? As opposed to "Definition of a UBO"? Which is essentially what this section is about?

Author: "Ownership of capital" is the term used in the API (type: HasCapital) and in the onboarding form. It distinguishes this UBO type from the legal representative UBO. Renaming it to "Definition of a UBO" would lose that distinction and create inconsistency with the API.

Reviewer: Make the description more plain english then as the way its currently written doesn't explain what a UBO is but focuses mainly on the API type - and the reader at this point hasn't been introduced to UBO's yet.

Author: Fair point, I'll rephrase to introduce the concept before referencing the API type!

> An **ownership of capital UBO** is anyone holding more than 25% of that company's value, whether directly or indirectly.

Author: ```suggestion
An ultimate beneficial owner (UBO) is any individual who ultimately owns or controls a company. The most common type is an ownership of capital UBO: anyone holding more than 25% of that company's value, whether directly or indirectly.
```

> An **ownership of capital UBO** is anyone holding more than 25% of that company's value, whether directly or indirectly.

Reviewer: Make this more plain english - this essentially defines what a UBO is.

Author: The section already defines both UBO types with plain English explanations. Rewriting it further risks losing the precision that distinguishes direct vs. indirect ownership, which is a compliance requirement. Out of scope for an archive PR regardless.

:::tip 
With the API, set `type` to `LegalRepresentative`.
On the onboarding form, choose `Legal representative` for type of control exercised.
No other additional information is required in the UBO section for legal representative UBOs.
:::

### Representatives {#representatives}

Representatives are individuals or legal entities authorized to act on behalf of the company during the onboarding process and for ongoing account management.
This is **separate from the legal representative** who serves as the account admin and undergoes identification verification.

Swan supports two types of representatives:

- **Individual representatives**: natural persons authorized to represent the company.
- **Company representatives**: legal entities authorized to represent the company.

Representative data is **optional** and helps Swan automate verification processes and reporting.

:::tip API usage
Use the `representatives` field in the `onboardCompanyAccountHolder` and `updateCompanyOnboarding` mutations.
Set `type` to `Individual` for natural persons or `Company` for legal entities.
:::

### Retrieving data automatically {#retrieve-data}

> For companies located in 🇫🇷 France, Swan retrieves and prefills available data automatically from the **National Business Register (RNE)** after you provide the company's registration number.

Reviewer: I believe its not just France that we retrieve information from.

Author: France (RNE) is the only country where Swan retrieves data automatically at the time this content was written. If this has changed, it needs to be verified with the product team and updated in the live docs first, then reflected in the archive. Out of scope for this PR.

Reviewer: Right - its been updated to read better in the new update.

|`residencyAddress.city`| <Req /> | <Req /> | <Req /> | <Req /> | <Req /> | <Req /> |
|`residencyAddress.country`| <Req /> | <Req /> | <Req /> | <Req /> | <Req /> | <Req /> |
|`residencyAddress.postalCode`| <Req /> | <Req /> | <Req /> | <Req /> | <Req /> | <Req /> |
|`residencyAddress.state`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
|`taxIdentificationNumber`| <Opt /> | <Opt /> | <Req /> ∗ |<Req /> | <Opt /> | <Cnd /> ∗∗ |
|`tradeName`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
|`typeOfRepresentation`| <Req /> | <Req /> | <Req /> | <Req /> | <Req /> | <Req /> |
|`vatNumber`| <Opt /> | <Opt /> | <Opt /> | <Req /> | <Opt /> | <Opt /> |

> ∗ *If the `accountCountry` is Germany, the account holder has 90 days to provide their Tax ID Number when opening an account.
Otherwise, the account is `Suspended`.
Share [Swan's Support Center article](https://support.swan.io/hc/en-150/articles/21876111550621-German-Tax-identification-number-Tax-ID) if your account holders have concerns.*
>
> ∗∗ *Required if **both** the `accountCountry` and `residencyAddress.country` are Spain.*

> :::info

Reviewer: This needs a better heading.

> :::info

Author: ```suggestion
:::info Registration requirements
```

*`isRegistered` must be `true` and the `registrationNumber` provided for **all onboardings** where `companyType` is `Company`.
This also applies when `companyType` is `SelfEmployed` **and** the account country is `France`.
Swan overrides `isRegistered` if the provided value is `false` but it should be `true`.
Learn more about how [Swan retrieves data](#retrieve-data) for certain company accounts.*
:::

> ### UBO information {#country-reqs-ubo}

Reviewer: This needs a better heading.

Reviewer: Is there a way to freeze the columns? With the long tables its difficult to read the lines and match them to the country.

Reviewer: In general - all the tables are very difficult to parse through. Can we simplify how this information is presented.

Reviewer: Im assuming that this will be solved in the v2 version of this.

Author: Agreed the tables are dense, this is a known limitation of the v1 API structure. The v2 docs will address this. Reformatting the tables is out of scope for an archive PR.

    |`representatives.roles`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
    |`representatives.entityName`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
    |`representatives.registrationNumber`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
    |`representatives.legalForm`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
    |`representatives.residencyAddress.addressLine1`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
    |`representatives.residencyAddress.addressLine2`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
    |`representatives.residencyAddress.city`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
    |`representatives.residencyAddress.country`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
    |`representatives.residencyAddress.postalCode`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
    |`representatives.residencyAddress.state`| <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> | <Opt /> |
  </TabItem>
</Tabs>

> ## User flow diagrams {#diagrams}

Reviewer: Can we update the embed so it pulls a zoomed out view of the diagrams? It currently focuses on the wrong part.

Author: The Figma URL parameters would need to be updated by whoever owns the Figma file. Out of scope for this PR. I also suggest we refrain from using Figma files in the future.

Reviewer: We own the Figma file. The file is also not up to date and needs to be updated as part of this onboarding work.

Author: How about we start a ticket to improve the Figma flow directly for v2?

If I'm the assignee for the Figma ticket, I'd like to seize the opportunity to move away from the Figma format entirely, into a proper docs-as-code approach for scalability and maintainability. Does that sound like a good idea?

Author: Opened: https://linear.app/swan/issue/DOC-1692/replace-figma-user-flow-diagrams-with-docs-as-code-for-onboarding-v2
